### PR TITLE
Add responsible computing

### PR DESCRIPTION
**Pull request by mhagdorn**
found some teaching resources for integrating ethics into computing

### CODE REVIEW

File: collections/_resources/responsible-computing.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Responsible Computing
+link: https://www.internetruleslab.com/responsible-computing
+skills: [TEACH,TEAM,USERS,MOD,SRU,NEW]
```
Comment (MakisH) [inline collections/_resources/responsible-computing.md:4]: Thank you, looks relevant!
Could you please comment on why you chose these particular skills? (this is an issue we have with all resources)

Comment (mhagdorn) [reply collections/_resources/responsible-computing.md:4]: I must admit a struggled with that. I figured the communications skills TEACH, TEAM, USERS fit quite nicely. ethical considerations are required for NEW and MOD. SRU probably requires that as well. Although that's the one I am least convinced by.

Comment (MakisH) [reply collections/_resources/responsible-computing.md:4]: I trust you, I just wanted to ensure a double-check.

Comment (mhagdorn) [reply collections/_resources/responsible-computing.md:4]: no worries and good call

**Pull request merged**